connect/transforms/src/test/java/org/apache/kafka/connect/transforms/CastTest.java

```diff
@@ -74,6 +79,7 @@ public void castWholeRecordKeyWithSchema() {
 
         assertEquals(Schema.Type.INT8, transformed.keySchema().type());
         assertEquals((byte) 42, transformed.key());
+        xform.close();
```

Comment:
Any reason we don't use the "try with resources" in these other cases as well? Maybe since the constructor has no arguments, we can just use a global instance and close in an `@After` method? Same for the other transforms below.